docs/reference-guides/block-api/block-supports.md

```diff
@@ -543,6 +543,79 @@ supports: {
 }
 ```
 
+## layout
+
+-   Type: `boolean` or `Object`
+-   Default value: null
+-   Subproperties:
+    -   `default`: type `Object`, default value null
+    -   `allowSwitching`: type `boolean`, default value `false`
+    -   `allowEditing`: type `boolean`, default value `true`
+    -   `allowInheriting`: type `boolean`, default value `true`
+    -   `allowSizingOnChildren`: type `boolean`, default value `false`
+    -   `allowVerticalAlignment`: type `boolean`, default value `true`
+    -   `allowJustification`: type `boolean`, default value `true`
+    -   `allowOrientation`: type `boolean`, default value `true`
+
+This value only applies to blocks that are containers for inner blocks. If set to `true` the layout type will be `flow`. For other layout types it's necessary to set the `type` explicitly inside the `default` object.
+
+### layout.default
+
+-   Type: `Object`
+-   Default value: null
+
+Allows setting the `type` property to define what layout type is default for the block, and also default values for any properties inherent to that layout type, e.g., for a `flex` layout, a default value can be set for `flexWrap`.
```

Comment:
>  also default values for any properties inherent to that layout type, e.g., for a `flex` layout, a default value can be set for `flexWrap`.

No biggie, but just curious about whether we should provide illustrative examples since theme.json doesn't cover any of the possible extended values?

```json
			"default": {
				"type": "flex",
				"flexWrap": "nowrap"
			}
```

Reply:
hmm good point perhaps we should add all the possible values to the schema, or they might erroneously be flagged as invalid